packages/select/src/vaadin-select-value-button.js

```diff
@@ -3,17 +3,17 @@
  * Copyright (c) 2017 - 2022 Vaadin Ltd.
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
-import { Button } from '@vaadin/button/src/vaadin-button.js';
+import { html, PolymerElement } from '@polymer/polymer/polymer-element.js';
+import { ButtonMixin } from '@vaadin/button/src/vaadin-button-mixin.js';
+import { ThemableMixin } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 import { css, registerStyles } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 
 registerStyles(
```

Comment:
The styles defined with `registerStyle` could be moved to the `<style>` tag of the template. Not sure if I see any reason to keep them in a separate style module.